test/parallel/test-https-server-keep-alive-timeout.js

```diff
@@ -59,13 +55,9 @@ test(function serverKeepAliveTimeoutWithPipeline(cb) {
 });
 
 test(function serverNoEndKeepAliveTimeoutWithPipeline(cb) {
-  let requestCount = 0;
-  process.on('exit', () => {
-    assert.strictEqual(requestCount, 3);
-  });
-  const server = https.createServer(serverOptions, (req, res) => {
-    requestCount++;
-  });
+  const server = https.createServer(
+    serverOptions,
+    common.mustCall((req, res) => {}, 3));
```

Comment:
thx, I have already simplify common.mustCall.this made test more readable. @jasnell